applications/TrilinosApplication/custom_strategies/builder_and_solvers/trilinos_block_builder_and_solver.h

```diff
@@ -4,954 +4,868 @@
-        TSystemVectorType& Dx,
-        TSystemVectorType& b) override
+    void InitializeSolutionStep(ModelPart& rModelPart,
+                                TSystemMatrixType& A,
+                                TSystemVectorType& Dx,
+                                TSystemVectorType& b) override
     {
         KRATOS_TRY
         KRATOS_CATCH("")
     }
 
     //**************************************************************************
     //**************************************************************************
-    void FinalizeSolutionStep(
-        ModelPart& r_model_part,
-        TSystemMatrixType& A,
-        TSystemVectorType& Dx,
-        TSystemVectorType& b) override
+    void FinalizeSolutionStep(ModelPart& rModelPart,
+                              TSystemMatrixType& A,
+                              TSystemVectorType& Dx,
+                              TSystemVectorType& b) override
     {
     }
 
-
     //**************************************************************************
     //**************************************************************************
-    void CalculateReactions(
-        typename TSchemeType::Pointer pScheme,
-        ModelPart& r_model_part,
-        TSystemMatrixType& A,
-        TSystemVectorType& Dx,
-        TSystemVectorType& b) override
+    void CalculateReactions(typename TSchemeType::Pointer pScheme,
+                            ModelPart& rModelPart,
+                            TSystemMatrixType& rA,
+                            TSystemVectorType& rDx,
+                            TSystemVectorType& rb) override
     {
+        TSparseSpace::SetToZero(rb);
 
-        TSparseSpace::SetToZero(b);
+        // refresh RHS to have the correct reactions
+        BuildRHS(pScheme, rModelPart, rb);
 
-        //refresh RHS to have the correct reactions
-        BuildRHS(pScheme, r_model_part, b);
-
-        //initialize the Epetra importer
-        // TODO: this part of the code has been pasted until a better solution is found
-        int system_size = TSparseSpace::Size(b);
+        // initialize the Epetra importer
+        // TODO: this part of the code has been pasted until a better solution
+        // is found
+        int system_size = TSparseSpace::Size(rb);
         int number_of_dofs = BaseType::mDofSet.size();
-        std::vector< int > index_array(number_of_dofs);
+        std::vector<int> index_array(number_of_dofs);
 
-        //filling the array with the global ids
+        // filling the array with the global ids
         int counter = 0;
-        for(typename DofsArrayType::iterator i_dof = BaseType::mDofSet.begin(); i_dof != BaseType::mDofSet.end(); ++i_dof)
-        {
-            int id = i_dof->EquationId();
-            if( id < system_size )
-            {
+        int id = 0;
+        for (const auto& dof : BaseType::mDofSet) {
+            id = dof.EquationId();
+            if (id < system_size) {
                 index_array[counter] = id;
```

Comment:
```suggestion
                index_array[counter++] = id;
```
then you can delete the next line